charts/lfx-platform/values.yaml

```diff
@@ -14,12 +14,28 @@ lfx:
 # Traefik configuration
 traefik:
   enabled: true
-  # Additional traefik configuration can be added here
-  # Values will be merged with the subchart's values
-  ingressRoute:
-    dashboard:
+  providers:
+    # Enable Kubernetes Gateway provider
+    kubernetesGateway:
+      enabled: true
+    # Disable Kubernetes Ingress provider
+    kubernetesIngress:
+      enabled: false
```

Comment:
is this necessary/recommended? if so, should we also disable kubernetesCRD, since we wouldn't be using it either?

and, not on this line, but I'd also like to see us remove the `authelia.ingress` settings from our default values file, since we are creating the resources ourselves, now.